### PR TITLE
Docs: Update Telegram Documentation

### PR DESCRIPTION
**Pull request by sintaherena**
# Monika Pull Request (PR)

## What feature/issue does this PR add

1. Update documentation telegram  https://monika.hyperjump.tech/tutorial/run-using-telegram-notification#_top

## How did you implement / how did you fix it

1. Update dependency
2. Update documentation telegram

## How to test

1. Run the application
2. Open the URL: http://localhost:3000
3. Select the documentations menu
4. Select the Run with Telegram Notification in Sidebar

Image:

### CODE REVIEW

**Review by sapiderman — APPROVED** [review]

File: docs/src/pages/tutorial/run-using-telegram-notification.md [hunk 1]
```diff
@@ -20,9 +20,13 @@ Now the next step is to create a group for the Monika notifications. Create a gr
 
 ![](https://miro.medium.com/max/1400/1*jI767UEJfnefsev7JDILOQ.png)
 
-Now that we have our bot API key and set our bot to be a group administrator, all we need to do is to get the group ID. Invite a new bot to the group called **@RawDataBot** into your group to get the group ID.
+Now that we have our bot API key and set our bot to be a group administrator, all we need to do is to get the group ID. Invite one of the following bots into your group to obtain the group ID:
```
Comment (nico-hyperjump) [inline docs/src/pages/tutorial/run-using-telegram-notification.md:23]: all we need to do **is get** the group ID

File: docs/src/pages/tutorial/run-using-telegram-notification.md [hunk 2]
```diff
@@ -20,9 +20,13 @@ Now the next step is to create a group for the Monika notifications. Create a gr
...
 
-![](https://miro.medium.com/max/762/1*FlZ_WX2zPPqmvpQNY9GWmA.png)
+- [getmyid_bot](https://t.me/getmyid_bot)
+- [getidsbot](https://t.me/getidsbot)
+- [RawDataBot](https://t.me/RawDataBot)
+
```
Comment (nico-hyperjump) [inline docs/src/pages/tutorial/run-using-telegram-notification.md:28]: If those bots are not active anymore, you need to find another bot that can obtain the group ID.

File: docs/src/pages/tutorial/run-using-telegram-notification.md [hunk 3]
```diff
@@ -20,9 +20,13 @@ Now the next step is to create a group for the Monika notifications. Create a gr
...
+![](https://miro.medium.com/v2/resize:fit:640/format:webp/1*uN_ddeb5OT649Jrf0WXm6A.png)
 
 After you have successfully invited the Telegram Bot Raw, you will see the group ID. **Save the group ID to somewhere else** and don’t forget to **kick the Telegram Bot Raw from your group**. Now that we have the group ID and bot token, it’s time to integrate it with Monika.
```
Comment (nico-hyperjump) [inline docs/src/pages/tutorial/run-using-telegram-notification.md:31]: After successfully inviting the Telegram Bot Raw, you’ll see the group ID. **Save this ID somewhere safe** and **remove the bot from your group**. Now that we have the group ID and bot token, we can proceed to integrate it with Monika.

**Review by nico-hyperjump — CHANGES_REQUESTED** [review]

File: docs/src/pages/tutorial/run-using-telegram-notification.md (hunk 3 above)
Comment (sintaherena) [reply docs/src/pages/tutorial/run-using-telegram-notification.md]: updated

image

**Pull request merged**